Core/include/Acts/Geometry/detail/DefaultGeometryIdGetter.hpp

```diff
@@ -0,0 +1,45 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/GeometryID.hpp"
+
+namespace Acts {
+namespace detail {
+
+/// Uniform access to geometry identifiers for different object types.
+struct DefaultGeometryIdGetter {
+  // explicit geometry identifiers are just forwarded.
```

Comment:
This is my thinking here: the `struct` is not part of the public interface since it is in the `detail` namespace. The structure itself is already documented as a courtesy. The comments for each function are not intended for the user but for me if I need to modify something in the future.